docs/book/src/getting-started/usage.md

```diff
@@ -47,15 +47,15 @@ foo
 
 ## [OPTIONAL] Sync with Kubernetes Secrets
 
-Refer to [Sync as Kubernetes Secret](topics/sync-as-kubernetes-secret.md) for steps on syncing the secrets-store content as Kubernetes secret in addition to the mount.
+Refer to [Sync as Kubernetes Secret](https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/master/docs/book/src/topics/sync-as-kubernetes-secret.md) for steps on syncing the secrets-store content as Kubernetes secret in addition to the mount.
```

Comment:
```suggestion
Refer to [Sync as Kubernetes Secret](../topics/sync-as-kubernetes-secret.md) for steps on syncing the secrets-store content as Kubernetes secret in addition to the mount.
```

We only need the relative path for the docs website to redirect to correct page.

Reply:
sure, updated.